edge/pkg/devicetwin/dtmanager/communicate.go

```diff
@@ -123,10 +124,9 @@ func dealConfirm(context *dtcontext.DTContext, resource string, msg interface{})
 }
 
 func detailRequest(context *dtcontext.DTContext, msg interface{}) (interface{}, error) {
-	//todo eventid uuid
 	getDetail := dttype.GetDetailNode{
 		EventType: "group_membership_event",
-		EventID:   "123",
+		EventID:   uuid.NewV4().String(),
```

Comment:
Double-check, now `EventID` is useless?

Reply:
the EventID of GetDetailNode struct now will become the content of message, no other use.